android/app/src/main/java/app/organicmaps/routing/RoutingController.java

```diff
@@ -905,12 +862,6 @@ public void setRouterType(@Framework.RouterType int router)
       build();
   }
 
-  @Framework.RouterType
-  public int getLastRouterType()
```

Comment:
How can AA check that the already built route on the mobile phone is of type VEHICLE?

Reply:
I've reviewed the logic, and this function is not necessary. Calling `setRouterType` is sufficient.

Reply:
Ah, we can keep this method if it is needed for AA. I just removed all unused methods highlighted by Android Studio,

Reply:
Restored.